OxfordBodleian/Juel-Jensen/BDLaethg33.xml

```diff
@@ -490,27 +490,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         ወይከልልዎ<supplied reason="omitted">፡</supplied> ለመድ<supplied reason="omitted">ኃ</supplied>ኔ፡ ዓለም፡ ሥጋሁ፡ ወደሙ፡ ለምድኃኔ፡ ዓለ፡
                                         ወንብጻሕ፡ ቅድመ፡ ገ<gap reason="illegible" unit="chars" quantity="1"/>፡ ለመድኃኔ፡ ዓለም፡ በአሚነ፡ ዚአሁ፡ ለክርስቶስ፡ ንገኒ።</q>
                                 </item>
+                            
                                 <item xml:id="e1">
-                                    <desc>A white sticker is glued on <locus target="#1r"/> with the signature "Ms 14" printed on it.
-                                    </desc>
-                                    <!--quotation marks-->
+                                    <desc>Scribal corrections are rare and mostly consist of letters or words written over erasures (e.g., <locus target="#41v #70r #78r"/>),
```

Comment:
Just one quick question: should a comma be added after e.g.?
We haven't been uniform in this. If there's a rule, I might standardise it in `handDesc` as well

Reply:
Great thank you. This will be very helpful to me for the revision of the handDesc part, since it is rife with examples from abbreviations, palaeographical features, etc.